packages/did-method-element/docs/local-test.md

```diff
@@ -18,8 +18,10 @@ set up and run and Ethereum node locally.
 
 - Install Dependencies
 - Clone and Build the Repository
-- Start Required Services
+- Create and Fund a Wallet
```

Comment:
consider a better name for this file than "local-test.md"

Reply:
`local-element-ropsten-install.md`?